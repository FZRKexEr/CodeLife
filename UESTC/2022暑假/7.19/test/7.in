10
ctgmohyvomnhwvayahovzmurwxnbpjchzyolpxduuabpyyefupixdhdvzxwcsyfpolhfntnptqbmrwehhvizvmmlnruhaygmckkg
3
egeetzomyiznkjxtrgeypachbekqukonqipnrpehbtdkgfopfbxsbijxowcwigihxbrixplmoxkznchyvatnoymnuaoyicksxtdv
1
xudgkjsdlxcxhihfiuzjjywfscneftpdwnlsrdrqlshnmbhylpxzvkfmzggnqoplfbyyhongmjjofwaaewwecskbrlkylqchggfp
9
bxmqxcnphtannsiuqsrsvsjqldktwxkdwavtlfufjjueesqzzqtdgyqfdsgxhiswiiidjhioedoqsafjiffkmxjsxezhqanymypx
2
jzeowpxssqbglmenpbmdxvppighlarbxdthnxhjmejxwpyyyaywuvwoihpizpainwgewwbanwzgimkskwxeunpxxzcempsmtxojr
5
pqxwbcwxpzuyvzvbvamsoyflbmpyhckdxczvebqnudmqscjvyohseiwabypejuqyxkhrhjgftwdxighjqufiagywjljbjodqpwwp
8
vffijzxiagfrnloqojxszcliqyguvifetsmkrcxdhwhtzvdtvyybujkwsydwgelcvxwefcobyknmffcficllebncxmjyhjcxlfsk
4
bswurnqqmeskrriibwyhyismrfvaruefcsjcvscljfciiruosqywsxgbokwknmifbgpxfibgiwoqlcpqkoktbmjkxvxxsodkahjn
7
xzvbshblepjereiznvvmmefjhebupsuolsvtzbdnhhyljkepqkfsmdawyeycelhqxkaynduqlicwzespmkihkaithjrafzlhjrfv
7
rhruglygszunqjafoyunwvwimhzcrkcdfeyolwpcmvbczaomasbynuhjbhcizupehldnlsyqkbuhwrnukwskbsolwbgpfrxjlkkv
1
