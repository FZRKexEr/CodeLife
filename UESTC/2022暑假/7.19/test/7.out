603979768
882499718
2048
65980984
1048576
4096
33554432
16384
16384
882499718
